mapactionpy_arcmap/MapReport.py

```diff
@@ -16,28 +16,29 @@ def dump(self):
         for report in self.results:
             if (reportIter > 0):
                 resultJson = resultJson + ","
-            resultJson = resultJson + report.toJSON()    
-            if (report.added == False):
+            resultJson = resultJson + report.toJSON()
+            if not report.added:
                 failCount = failCount + 1
             reportIter = reportIter + 1
 
         resultJson = resultJson + "]}"
 
         if (reportIter == 0):
-            self.result = "Failure";
+            self.result = "Failure"
             self.summary = "No layers provided in recipe for '" + self.productName + "' product."
         else:
             if (failCount == 0):
                 self.result = "Success"
                 self.summary = "'" + self.productName + "' product generated successfully."
             else:
                 self.result = "Warning"
-                self.summary = str(failCount) + " / " + str(reportIter) + " layers could not be added to '" + self.productName + "' product."
+                self.summary = str(failCount) + " / " + str(reportIter) + \
+                    " layers could not be added to '" + self.productName + "' product."
 
         resultJson = "{\"result\":\"" + self.result + \
-                        "\", \"productName\":\"" + self.productName + \
-                        "\", \"summary\":\"" + self.summary + \
-                        "\", \"classification\":\"" + self.classification + \
-                        "\", \"results\":[" + resultJson
+            "\", \"productName\":\"" + self.productName + \
```

Comment:
Please use a proper json library to output this. `jsonpickle` works nicely and can added as a dependency for this module if required.